src/main/java/sg/ncl/MainController.java

```diff
@@ -3910,9 +3912,16 @@ private Realization extractRealization(String json) {
                 Map<String, String> nodeDetails = new HashMap<>();
                 String nodeName = (String) key;
                 JSONObject nodeDetailsJson = new JSONObject(nodesInfoObject.get(nodeName).toString());
-                nodeDetails.put("os", nodeDetailsJson.getString("os"));
-                nodeDetails.put("qualifiedName", nodeDetailsJson.getString("qualifiedName"));
-                nodeDetails.put(NODE_ID, nodeDetailsJson.getString(NODE_ID));
+                log.info("Key is: {}", nodeName);
+//                nodeDetails.put("os", nodeDetailsJson.getString("os"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)